nimble/host/audio/include/audio/ble_audio.h

```diff
@@ -527,6 +530,24 @@ struct ble_audio_broadcast_name {
 /** BLE Audio event: Codec Unregistered */
 #define BLE_AUDIO_EVENT_CODEC_UNREGISTERED                   2
 
+/** BLE Audio event: BASS - Remote Scan Stopped */
+#define BLE_AUDIO_EVENT_BASS_REMOTE_SCAN_STOPPED             3
+
+/** BLE Audio event: BASS - Remote Scan Started */
+#define BLE_AUDIO_EVENT_BASS_REMOTE_SCAN_STARTED             4
+
+/** BLE Audio event: BASS - Add Source */
+#define BLE_AUDIO_EVENT_BASS_ADD_SOURCE                      5
+
+/** BLE Audio event: BASS - Modify Source */
+#define BLE_AUDIO_EVENT_BASS_MODIFY_SOURCE                   6
+
+/** BLE Audio event: BASS - Set Broadcast Code */
+#define BLE_AUDIO_EVENT_BASS_SET_BROADCAST_CODE              7
+
+/** BLE Audio event: BASS - Remove Source */
+#define BLE_AUDIO_EVENT_BASS_REMOVE_SOURCE                   8
```

Comment:
```suggestion
/** BLE Audio event: BASS - Add Source */
#define BLE_AUDIO_EVENT_BASS_SOURCE_ADDED                      5

/** BLE Audio event: BASS - Modify Source */
#define BLE_AUDIO_EVENT_BASS_SOURCE_MODIFIED                   6

/** BLE Audio event: BASS - Set Broadcast Code */
#define BLE_AUDIO_EVENT_BASS_SET_BROADCAST_CODE              7

/** BLE Audio event: BASS - Remove Source */
#define BLE_AUDIO_EVENT_BASS_SOURCE_REMOVED                   8
```

TBH, if all the events carry status only, we could end up with having just 1 event:
```
#define BLE_AUDIO_EVENT_BASS_OPERATION_STATUS           X

/** @brief BASS operation status */
struct ble_audio_event_bass_op_status {
    /** Source ID */
    uint8_t source_id;

   /** Operation */
   uint8_t op;

    /** Event status. 0 on success, BLE_HS Error code otherwise */
    uint8_t status;
};
```

Reply:
I agree, but what about `remote_scan_stopped`, `remote_scan_started` and `bass_set_broadcast_code`? these dont conform to the format - scan ones have conn handle, and no status, and broadcast code is extra field. Should I replace all the other ones? But then, we have `type = BLE_AUDIO_EVENT_BASS_OPERATION_STATUS` for some, and additional OP (define or enum) to differentiate them.